src/HomeWork/Sum/SumMain.java

```diff
@@ -1,33 +1,33 @@
 package HomeWork.Sum;
 
-import HomeWork.NumbRevers.ReversMethod;
-
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class SumMain {
     public static void main(String[] args) {
         boolean miss = false;
```

Comment:
default value of primitive boolean is false, there is no need to initialize it

it is better to start the name of boolean variables with is... (isExit or smt like this)

Reply:
done